### DISCUSSION TITLE
How to add custom country and sectors

### DISCUSSION THREAD

**Discussion by ziritione85** [ghostfolio/ghostfolio#3681] [Q&A]
So, basically how I can add to a custom item the countries and the sectors. Found nothing. Thanks!

**Comment by dtslvr** [comment]
Thank you for your suggestion, @ziritione85! You can change countries and sectors directly in the database.

The idea to edit this via the GUI has been proposed in https://github.com/ghostfolio/ghostfolio/discussions/2619. To keep everything organized, we are going to close this discussion and continue the conversation there.

Please feel free to follow the progress and share any additional insights or information on the original discussion thread. Your input is always welcome!
